README.md

```diff
@@ -1 +0,0 @@
-# ScalaIntership
```

Comment:
Need more details about 
1) what program does
2) how to run the program
3) how it could be configured